conda/resolve2.py

```diff
@@ -0,0 +1,905 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Is this supposed to be here? Seems like just one of those work files I often check in accidentally